x-pack/plugins/observability_ai_assistant/common/functions/transform.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import dedent from 'dedent';
+import { FromSchema } from 'json-schema-to-ts';
+
+export const transformFunctionDefinition = {
+  name: 'transform_create',
+  contexts: ['core'],
+  description:
+    dedent(`Use this function to create Elasticsearch transforms. This function does not return data to the assistant, it only shows it to the user.
+
+    Ask the user if they want to provide an ingest pipeline, if they do, use that, otherwise, omit pipeline in the request.
```

Comment:
When testing I wasn't asked about this, perhaps we need to word this strong, `You MUST ask`